### ISSUE TITLE
How to reset the learning rate when finetuning model 

### ISSUE THREAD

**Issue by 282866787** [tensorflow/tensor2tensor#930]
### Description
I met a problem when I trained my En-Zh model, that I want to sample a part of training data to fine tune my model wihin 5 epochs. Due to the large training steps = 2300000 , the learning rate is so small (2e-5) that I can't get significant experimental phenomena which I do right or not. I want to gain a method that I can reset the learning rate about 5e-5 or even larger, does anyone have tips ?
...

### Environment information

```
OS: <your answer here>

$ pip freeze | grep tensor
# your output here
pip install tensorflow-gpu==1.6
tensor2tensor-1.6.5
$ python -V
# your output here
```3.6.5

### For bugs: reproduction and error logs

```
# Steps to reproduce:
...
```

```
# Error logs:
...
```

**Comment by martinpopel** [comment]
> the learning rate is so small (2e-5) that I can't get significant

First, I am not sure if this is the cause. I found out that finetuning works even with a very small learning rate works. That said, there are three ways how to increase the learning rate for finetuning:

* Just increase the parameter `learning_rate_constant` (or `learning_rate` in older T2T versions).

* You can define your own learning rate schedule by modifying the source code. E.g. that after 2.3M steps the learning rate will go up (as in warmup) and then down (decay) in whatever pace you want. If you don't like coding, you can use piecewise-constant schedule and just provide the constants (but it seems sudden big jumps in learning rate are not good for the learning stability).

* You can also reset the `global_step` variable stored in the checkpoint to 0. This way the learning rate schedule will start from scratch with the warmup. My experience is that after the warmup the learning rate is so high that it effectively ruins whatever the model has learned so far, so I don't recommend this option for finetuning, but you can set the global_steps to non-zero value. If you run the avg_checkpoints.py script with just one checkpoint, it will reset the global_step to 0 (and you can easily tweak it to reset it to another value).
